db/migrations/sqls/20200526142326-add-refresh-tokens-up.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE tokens (
```

Comment:
Do we want to be explicit here and call this refresh_tokens or should we stick with the more generic tokens which doesn't assume any specific implementation ?

Reply:
refresh_tokens make more sense, changing it.